src/engine/framework/Rcon.cpp

```diff
@@ -47,8 +47,10 @@ Message::Message( const netadr_t& remote, std::string command,
 {}
 
 Message::Message( std::string error_message )
-	: error(std::move(error_message))
-{}
+	: secure(Secure::Invalid), error(std::move(error_message))
```

Comment:
Not sure about this one. Uninitialized struct members access is UB, looks like `secure` set to `Secure::Invalid` will at least help to catch not set `secure` member later.

@slipher, @illwieckz Could you please take a look?